docs/source/documentation/data-products/components/product-specification.html.md.erb

```diff
@@ -96,7 +112,9 @@ The product `title` should be the human-readable version of the name. Keep it sh
 
 The product `description` can be longer, give more information and will help users find your data.
 
-> NOTE: the `product-version` attribute is not yet documented here. This will be automatically incremented when the Data Product is updated. Versioning logic will be clearly documented, but in brief breaking changes will constitute Major version bumps (`v1.0` -> `v2.0`), where backwards-compatible changes and updates will be Minor version bumps (`v1.0` -> `v1.1`).
+> NOTE: the `product-version` attribute is not yet documented here. This will be automatically incremented when the Data Product is updated. [Versioning logic is documented here](/documentation/data-products/index.html#altering-a-data-product), but in brief breaking changes will constitute Major version bumps (`v1.0` -> `v2.0`), where backwards-compatible changes and updates will be Minor version bumps (`v1.0` -> `v1.1`).
```

Comment:
(minor copy suggestion)

```suggestion
> NOTE: the `product-version` attribute is not yet documented here. This will be automatically incremented when the Data Product is updated. [Versioning logic is documented here](/documentation/data-products/index.html#altering-a-data-product), but in brief, breaking changes will constitute Major version bumps (`v1.0` -> `v2.0`), whereas backwards-compatible changes and updates will be Minor version bumps (`v1.0` -> `v1.1`).
```